src/server/components/dashboard/__tests__/controllers.test.ts

```diff
@@ -268,33 +267,6 @@ describe("Dashboard Controllers", () => {
     });
   });
 
-
-  describe.only('calculateSortOrder', () => {
-    test('returns correct sort order when there are no query parameters', () => {
-      const expectedResult = [{ country: 'asc' }, { type: 'asc' }];
-      expect(calculateSortOrder({})).toEqual(expectedResult);
-    });
-
-    test("returns correct sort order when admins query parameter is not provided", () => {
```

Comment:
this test doesn't match up with what you're testing. I've changed in listsController.tests.ts